script.js

```diff
@@ -0,0 +1,124 @@
+var nouns = [];
+var adjectives = [];
+var adverbs = [];
+var verbs = [];
+console.log('nouns1:',nouns);
+var story;
+
+var backupNouns = ["parent", "car", "zebra", "incense"];
+var backupAdjectives = ["blue", "lovely", "grainy", "black"];
+var backupAdverbs = ["gracefully", "foolishly", "often", "obnoxiously"];
+var backupVerbs = ["burp", "walk", "swing", "scream"];
+
+function generateMadLib() {
+	nouns = [];
+	adjectives = [];
+	adverbs = [];
+	verbs = [];
+	var addNouns = function() {
+		var userNouns = document.getElementById('nouns').value;
+		userNouns = userNouns.split(",");
+		nouns = nouns.concat(userNouns);
+		nouns.shift();
```

Comment:
I remember we talked about the array.shift, as opposed to looping through and pushing each of these values one by one which is what most students did. I may have you talk to the class about this.